interfaces/IBF-dashboard/src/app/components/aggregates/aggregates.component.scss

```diff
@@ -8,6 +8,7 @@
 
   .aggregate-header-item-label {
     margin: 8px 0;
+    font-weight: 600;
```

Comment:
This is for the bold text, but that one is already reviewed according to DevOps, and I thought I saw it in a different PR already before?